docs/4.1/intro.md

```diff
@@ -32,7 +32,7 @@ environments, not servers. Below is the list of most popular Teleport features:
 Teleport is available through the free, open source edition ("Teleport Community Edition")
 or a commercial edition ("Teleport Enterprise Edition").
 
-![teleport diagram](/images/diagram-teleport.png)
+![teleport diagram](https://gravitational.com/images/diagram-teleport.png)
```

Comment:
Is it a good idea to use an absolute link here? Could cause confusion in future about why updates aren’t happening if the picture is updated locally and not pushed.

Reply:
This was intentional as this image is from the main marketing site vs the docs repo.